release_images_training.yml

```diff
@@ -64,3 +64,16 @@ release_images:
       example: False
       disable_sm_tag: True
       force_release: False
+  6:
+    framework: "huggingface_pytorch"
+    version: "2.3.0"
+    hf_transformers: "4.46.1"
+    arch_type: "x86"
+    training:
+      device_types: ["gpu"]
+      python_versions: [ "py311" ]
+      os_version: "ubuntu20.04"
```

Comment:
ubuntu 20 is eol in april, can we use a later os?
https://ubuntu.com/about/release-cycle

Reply:
The next HF image may bump this, but note we are not the author of the os version. JS need to work with HF to request for this upgrade